db/deploy/tables/applications.sql

```diff
@@ -5,10 +5,55 @@ begin;
 
 create table if not exists connectivity_intake_public.applications (
   id serial not null,
+  owner uuid not null,
   form_data jsonb,
   created_at timestamp with time zone default current_timestamp,
   updated_at timestamp with time zone default current_timestamp,
   primary key(id)
 );
 
+select connectivity_intake_public.upsert_timestamp_column('connectivity_intake_public', 'applications');
+
+create index connectivity_intake_owner_uuid on connectivity_intake_public.applications(uuid);
+
+do
+$grant$
+begin
+
+-- Grant connectivity_intake_auth_user permissions
+perform connectivity_intake_public.grant_permissions('select', 'applications', 'connectivity_intake_auth_user');
+perform connectivity_intake_public.grant_permissions('insert', 'applications', 'connectivity_intake_auth_user');
+perform connectivity_intake_public.grant_permissions('update', 'applications', 'connectivity_intake_auth_user'
+  ARRAY['first_name', 'last_name', 'email_address', 'created_at', 'created_by', 'updated_at', 'updated_by', 'deleted_at', 'deleted_by']);
+
+-- Grant connectivity_intake_guest permissions
+perform connectivity_intake_public.grant_permissions('select', 'applications', 'connectivity_intake_guest');
+
+end
+$grant$
+
+-- Enable row level security
+alter table connectivity_intake_public.applications enable row level security;
+
+do
+$policy$
+begin
+-- connectivity_intake_auth_user RLS: can see all users, but can only modify its own record
```

Comment:
this table isn't about users, and we don't want users to see all applications; we want users to see only their own applications and only modify their own applications